src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -47,6 +72,30 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
       response.sendError(400);
       return;
     }
-    // TODO: Use utcEncodings to generate Availabilities and store them in datastore.
+    UserService userService = UserServiceFactory.getUserService();
+    String email = userService.getCurrentUser().getEmail();
```

Comment:
Need to check if requestee == person logged in and return SC_UNAUTHORIZED (401)

Reply:
Not applicable because the request does not include info about the user.